client/my-sites/upgrades/domain-management/edit/card/header/index.jsx

```diff
@@ -21,16 +24,15 @@ const Header = React.createClass( {
 	},
 
 	render() {
-		const domain = this.props.domain;
+		const { domain, isDomainOnly } = this.props;
 
-		if ( ! domain ) {
+		if ( ! domain || isDomainOnly ) {
```

Comment:
If you look at the screenshots the `header` makes 0 sense for `domain-only` sites. It duplicates the header above it, and it was probably there only for the purpose of having that `Primary Domain`. :)